examples/language/openmoe/requirements.txt

```diff
@@ -0,0 +1,5 @@
+colossalai >= 0.1.12
```

Comment:
The version is too old. The latese release is `0.3.3`

Reply:
done